sklearn/metrics/tests/test_dist_metrics.py

```diff
@@ -76,6 +77,15 @@ def test_cdist(metric_param_grid, X, Y):
             # with scipy
             rtol_dict = {"rtol": 1e-6}
 
+        # TODO: Remove when scipy minimum version >= 1.7.0
+        # scipy supports 0<p<1 for minkowski metric >= 1.7.0
+        if metric == "minkowski":
+            # default value of p for Minkowski is 2
+            p = float(kwargs.get("p", 2))
+            print("p", p)
```

Comment:
In this test `p` should always be provided when `metric == "minkowski"` . Also we should avoid unnecessary printing.
```suggestion
            p = kwargs["p"]
```

Reply:
Oh forgot remove print. WIll make change